.github/workflows/bindings-java.yml

```diff
@@ -1,34 +1,28 @@
-name: Examine the Java Bindings
+name: Java bindings checks
 on:
   push:
-    branches:
-      - develop
-      - production
+    branches: [develop, production]
     paths:
-      - ".github/workflows/examine-java-bindings.yml"
+      - ".github/workflows/bindings-java.yml"
       - ".github/actions/**"
       - "client/bindings/java/**"
-      # Watch the Rust core too
-      - "client/src/**"
-      - "client/Cargo.toml"
-      - "Cargo.lock"
+      - "**.rs"
```

Comment:
With this is would also run if we change something in the Rust part of nodejs, no?

Reply:
Exclude sounds like the safer way

Reply:
Done